coops-ui/src/mixins/address-mixin.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Vue } from 'vue-property-decorator'
+import isEqual from 'lodash.isequal'
+
+/**
+ * Mixin that provides some useful address utilities.
+ */
+@Component
+export default class AddressMixin extends Vue {
+  /**
+   * Compares two address objects and returns a boolean indicating
+   * if they match.
+   *
+   * @param addressA An address object.
+   * @param addressB An address object.
+   * @return boolean True if they match, false if they do not.
+   */
+  isSameAddress (addressA: Object, addressB: Object): boolean {
+    return isEqual(addressA, addressB)
```

Comment:
So we found a problem with the "other" address comparison code: it didn't ignore the address type field. But the code sometimes worked because we weren't initializing the address type.
All this to say that I think we should initialize address type when we create the address objects, and the comparison functions will have to handle that.

Reply:
Okie doke, sounds good. Will look into that when i refactor and do the testing.

Reply:
As discussed earlier, we will delete/ignore addressType in the FE.

Do you need to do any further refactoring/testing or can I resolve this conversation?

Reply:
This work and refactor has been implemented in the follow up PR. Since we concluded to not initialize `addressType` and instead filter it out, the changes were separate from this particular work, so was included in the follow up PR.  I will make notes of these changes on PR #401